### DISCUSSION TITLE
Autocomplete in code editor

### DISCUSSION THREAD

**Discussion by deepakxyz** [nxt-dev/nxt_editor#127] [Ideas]
Autocomplete in the code editor would be very helpful and is there a way to implement vim key binding in the code editor.

**Comment by ImLucasBrown** [comment]
In general the code editor needs a polish pass. Its a complicated beast.
We're actually leaning towards making plugins for IDEs (PyCharm, VSCode, Sublime, ect) that would allow you to edit node code externally.
Autocomplete is on my list of things to get done regardless. The first version of it will probably be addressing #59 and then move on to completing variables. As of writing this I can't give a timeline for when that will happen.

As far as key bindings go, we do have a hotkey editor (Window > Hotkey Editor), but not everything has a hotkey and there isn't a way to import or select a key binding schema.

All that being said, we're always looking for collaborators, so if someone thinks they can take it on we'd gladly accept a pull request. :)
